tutorials/introductory/usage.py

```diff
@@ -473,10 +493,25 @@ def my_plotter(ax, data1, data2, param_dict):
 # :mod:`matplotlib` first tries to import it, if the import fails, it tries to
 # import `PySide`.
 #
+# Using non-builtin backends
+# --------------------------
+# 3rd parties can provide additional backends. Some of them (e.g. jupyter and
+# pycharm have their own way of activation). However, there is also a generic
+# way to activate non-builtin backends. If you have a module ``my_backend``
```

Comment:
"More generally, any importable backend can be selected by using any of the methods above with `module://name.of.the.backend` as the backend name, e.g. `matplotlib.use("module://name.of.the.backend")`."